newIDE/app/src/VariablesList/index.js

```diff
@@ -237,27 +287,49 @@ export default class VariablesList extends React.Component<Props, State> {
 
   _renderEmpty() {
     return (
-      <div>
-        <EmptyMessage
-          style={styles.emptyExplanation}
-          messageStyle={styles.emptyExplanationMessage}
-        >
-          {this.props.emptyExplanationMessage || ''}
-        </EmptyMessage>
-        <EmptyMessage
-          style={styles.emptyExplanation}
-          messageStyle={styles.emptyExplanationMessage}
-        >
-          {this.props.emptyExplanationSecondMessage}
-        </EmptyMessage>
-      </div>
+      !!this.props.emptyExplanationMessage && (
+        <div>
+          <EmptyMessage
+            style={styles.emptyExplanation}
+            messageStyle={styles.emptyExplanationMessage}
+          >
+            {this.props.emptyExplanationMessage}
+          </EmptyMessage>
+          <EmptyMessage
+            style={styles.emptyExplanation}
+            messageStyle={styles.emptyExplanationMessage}
+          >
+            {this.props.emptyExplanationSecondMessage}
+          </EmptyMessage>
+        </div>
+      )
     );
   }
 
   render() {
-    const { variablesContainer } = this.props;
+    const { variablesContainer, inheritedVariablesContainer } = this.props;
     if (!variablesContainer) return null;
 
+    // map all object variables, if they have been passed
```

Comment:
I would say:
`// Display parent variables, if any`

Display give more sense for the reader of the code, while map is just the operation that is written below

Reply:
The name should be containerInheritedVariablesTree to be consistent too ;)